extension/ezjscore/classes/ezjscpacker.php

```diff
@@ -515,9 +518,15 @@ static function fixImgPaths( $fileContent, $file )
                        $newMatchPath .= implode( '/', $cssPathSlice ) . '/';
                    }
                    $newMatchPath .= str_replace( '../', '', $match );
-                   $fileContent = str_replace( $match, $newMatchPath, $fileContent );
+                   // set match hash
+                   $hash = md5( uniqid( mt_rand(), true ) );
+                   // store hash and new match path
+                   $replace[$hash] = $newMatchPath;
+                   // replace match element by hash
```

Comment:
CS. (sorry for so many nitpicks) The existing comments all start with an upper case letter and I think it would be best to follow this convention within your additions.